08_voting/index.tex

```diff
@@ -1 +1,192 @@
+
+In this coursework, the pairwise Condorcet winner is chosen for a tiebreak setup which states that a candidate who dominates the other candidate in term of majority relation wins.
+
+% Whereas the Hare rule removes all candidates with the least first-place votes and Coombs Rule iteratively removes a candidate with most last placed votes.
+
+% Interestingly, Instant Runoff method is criticized by Condorcet for its disadvantage which involves removing a candidate who is favored by a majority of voters.
+
+\section{Voting Protocol}
+\label{sec:VotingProtocol}
+Voting Protocol is adopted from an example of formal specification given in Robert's Rules of Order (RONR) that defines a series of sequential actions when a voting event happens. This protocol is applicable to any voting method at certain state of the game (i.e. during IIGO session) whenever a voting event happens. With the basis from RONR\footnote{refer to Prof. J Pitt's Draft Book: Self-Organising Multi-Agent Systems (2020) Chapter 4 "Social Choice Theory", Section 4.6.1 Page 114}, the sequence of actions of voting protocol in this game when a voting event happens is as follows (taken an example of voting event for change of rules): the President selects a rule from the rule proposal list to be voted on; the President notifies the Speaker to hold a voting event for the selected rule; the Speaker calls for a vote for the selected rule by opening ballots to all islands to cast their votes; the Speaker closes the ballots after receiving votes from all islands; the Speaker counts the votes by calling an applicable voting method function for winner selection to produce a result; and finally the Speaker publicly declares the voting result whether the selected rule can be carried out or not. Figure~\ref{fig:RONRVotingProtocol} shows the general sequence of the voting protocol for rules.
+
+\begin{figure}[H]
+\begin{center}
+\begin{tikzpicture}[node distance=20pt]
+\centering
+\node[draw, rounded corners] (start)  {The President selects a rule to be voted on};
+\node[draw, below=of start] (step 1)  {The President notifies the Speaker to hold a voting session on the selected rule};
+\node[draw, below=of step 1] (step 2)  {The Speaker opens ballots for all islands to cast the votes};
+\node[draw, below=of step 2] (step 3)  {The Speaker closes ballots after receiving votes from all islands};
+\node[draw, below=of step 3] (step 4)  {The Speaker counts the votes by calling a voting method/function for winner selection};
+\node[draw, below=of step 4, rounded corners] (end)   {The Speaker publicly declares the result of voting};
```

Comment:
{The Speaker publicly declares the result **for the finalised vote.**

Reply:
changed